lib/components/shorts/short_scroll_view.dart

```diff
@@ -228,16 +215,11 @@ class ShortScrollView extends HookConsumerWidget {
       for (var i = 0; i < shortControllers.length; i++) {
         shortControllers[i].isCurrent = i == index % shortControllers.length;
       }
-      debugPrint('SHRT: setCurrentIndex: $index');
-      Future.delayed(300.ms, () {
```

Comment:
Husker ikke hvorfor. Veldig rart.
Det har hendt jeg putter in delays og conditions for å unngå UI lags ved scrolling og den type scenarioer, men tror ikke det var tilfellet her.